external/model-preparation-algorithm/configs/rotated-detection/efficientnetb2b_maskrcnn/template.yaml

```diff
@@ -1,10 +1,10 @@
 # Description.
-model_template_id: ClassIncremental_Instance_Segmentation_MaskRCNN_EfficientNetB2B
-name: MaskRCNN-EfficientNetB2B-ClsIncr
-task_type: INSTANCE_SEGMENTATION
+model_template_id: Custom_Rotated_Detection_via_Instance_Segmentation_MaskRCNN_EfficientNetB2B
+name: MaskRCNN-EfficientNetB2B
+task_type: ROTATED_DETECTION
 task_family: VISION
 instantiation: "CLASS"
-summary: Class-Incremental Instance Segmentation for MaskRCNN-EfficientNetB2B
+summary: Class-Incremental Rotated object detection for MaskRCNN-EfficientNetB2B
```

Comment:
Could you add NNCF entrypoint & base_model_path?

Reply:
I added it.